content/api/commands/mount.md

```diff
@@ -0,0 +1,226 @@
+
+To use `cy.mount` you will need to add a
+[custom command](/api/cypress-api/custom-commands) to the commands file. Below
+are examples that you can start with for your commands:
+
+:::react-vue-example
+
+```js
+import { mount } from '@cypress/react'
+Cypress.Commands.overwrite('mount', (jsx, options) => {
+  // Wrap any parent components needed
+  // ie: return mount(<MyProvider>{jsx}</MyProvider>, options)
+  return mount(jsx, options)
+})
+```
+
+```js
+import { mount } from '@cypress/vue'
+Cypress.Commands.overwrite('mount', (comp, options = {}) => {
+  // Setup options object
+  options.global = options.global || {}
+  options.global.stubs = options.global.stubs || {}
+  options.global.stubs['transition'] = false
+  options.global.components = options.global.components || {}
+  options.global.plugins = options.global.plugins || []
+
+  /* Add any global plugins */
+  // options.global.plugins.push({
+  //   install(app) {
+  //     app.use(MyPlugin);
+  //   },
+  // });
+
+  /* Add any global components */
+  // options.global.components['Button'] = Button;
+
+  return mount(comp, options)
+})
+```
+
+:::
+
+See [examples](#Examples) below for practical use cases.
+
+## Adding TypeScript Typings for `cy.mount` Commands
+
+When working in
+[TypeScript](https://docs.cypress.io/guides/tooling/typescript-support), you
+will need to add some custom typings for your commands to get code completion
+and avoid any TypeScript errors. Below are the typings for the above examples
+for React and Vue:
+
+:::react-vue-example
+
+```ts
+// cypress.d.ts
+import { MountOptions, MountReturn } from '@cypress/react'
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Mounts a React node
+       * @param jsx React Node to mount
+       * @param options Additional options to pass into mount
+       */
+      mount(
+        jsx: React.ReactNode,
+        options?: MountOptions
+      ): Cypress.Chainable<MountReturn>
+    }
+  }
+}
+```
+
+```ts
+import { mount } from '@cypress/vue'
+type MountParams = Parameters<typeof mount>
+
+declare global {
+  namespace Cypress {
+    type ComponentParam = MountParams[0] | JSX.Element
+    type OptionsParam = MountParams[1]
+    interface Chainable {
+      /**
+       * Helper mount function for Vue Components
+       * @param component Vue Component or JSX Element to mount
+       * @param options Options passed to Vue Test Utils
+       */
+      mount(component: ComponentParam, options?: OptionsParam): Chainable<any>
+    }
+  }
+}
+```
+
+:::
+
+The typings will need to be in a location that any code can access in the
+application. We recommend adding them to a `cypress.d.ts` file at the root of
+the application and updating any tsconfig.json files to reference the typings
+file in the `include` option:
+
+```json
+"include": ["./src", "cypress.d.ts"]
+```
+
+## Additional Mount Commands
+
+You can create additional mount commands to fit your needs. For instance, there
```

Comment:
I feel like "You can create additional mount commands to fit your needs." might be a little confusing for users. What about something like this?

> You're not limited to a single `cy.mount()` command. You can create any number of mount custom commands if needed, as long as they have unique names.